### PR TITLE
Scale cursor image

### PR DESCRIPTION
**Pull request by Shosoul**

### CODE REVIEW

File: UpdatingImage.lua [hunk 1]
```diff
@@ -29,9 +29,13 @@ function UpdatingImage:draw()
   -- note how the Quad's width and height are larger than the image width and height.
   local bg_quad = love.graphics.newQuad(x, y, self.width, self.height, self.image:getDimensions())
 
-  local scale = 1
+  local x_scale = 1
+  local y_scale = 1
   if not self.tiled then   
-    scale = self.width / math.max(self.image:getWidth(), self.image:getHeight()) -- keep image ratio
+    x_scale = self.width / self.image:getWidth()
+    y_scale = self.height / self.image:getHeight() -- keep image ratio
```
Comment (JamesVanBoxtel) [inline UpdatingImage.lua:36]: delete comment about ratio

File: graphics.lua [hunk 2]
```diff
@@ -946,12 +946,15 @@ end
 function Stack.render_cursor(self)
   local shake_idx = #shake_arr - self.shake_time
   local shake = ceil((shake_arr[shake_idx] or 0) * 13)
+  local scale_x = 40 / themes[config.theme].images.IMG_cursor[(floor(self.CLOCK / 16) % 2) + 1]:getWidth()
```
Comment (JamesVanBoxtel) [inline graphics.lua:949]: lets grab the image into a local variable so we don't duplicate the floor mod logic again below.

File: UpdatingImage.lua (hunk 1 above)
Comment (Shosoul) [reply UpdatingImage.lua:36]: Actually I just did in a different pull request lol

Comment (Shosoul) [reply UpdatingImage.lua:36]: oh wait nevermind

Comment (Shosoul) [reply UpdatingImage.lua:36]: #547

File: graphics.lua (hunk 2 above)
Comment (Shosoul) [reply graphics.lua]: Done

File: UpdatingImage.lua (hunk 1 above)
Comment (Shosoul) [reply UpdatingImage.lua:36]: I think I accidently built this pr off of that one.

**Review by JamesVanBoxtel — APPROVED** [review]

**Pull request merged**